job_spec/INSAR_ISCE.yml

```
@@ -93,7 +93,9 @@ INSAR_ISCE:
       cost: 1.0
   validators: []
   compute_environment:
-    name: 'Default'
+    name: 'Aria'
```

Should we perhaps make it more explicit that this compute environment only applies to this particular job type?
```suggestion
    name: 'InsarIsceAria'
```

---

That's a good idea!